.gitignore

```diff
@@ -22,3 +22,4 @@ _testmain.go
 *.exe
 
 .DS_Store
+*.swp
```

Comment:
Can you remove this from the `.gitignore`?

It is a detail from your editor's configuration. You set a global `.gitignore` on your machine so Git always ignores `.swp` files.